source/operations/data-recovery/recover-after-site-failure.rst

```diff
@@ -1,13 +1,95 @@
+
+Site Replication
+----------------
+
+:ref:`Site replication <minio-site-replication-overview>` keeps two or more MinIO deployments in sync with IAM policies, buckets, bucket configurations, objects, and object metadata.
+If a peer site fails, such as due to a major disaster or long power outage, you can use the remaining healthy site(s) restore the :ref:`replicable data <minio-site-replication-what-replicates>`.
+
+The following procedure can restore data in scenarios where :ref:`site replication <minio-site-replication-overview>` was active prior to the site loss.
+
+1. Deploy a new MinIO site using the same ``root`` credentials as used on other deployments in the site replication configuration
+
+   You can use the original hardware, if still available, but you must first wipe any remaining data before creating the new site.
+2. Configure the new site with the same Identity Provider (IDp) as the other sites
+3. :ref:`Expand the existing site replication <minio-expand-site-replication>` by adding the newly deployed site
+4. Remove the lost site from the site replication configuration
+
+Site replication healing automatically adds IAM settings, buckets, bucket configurations, and objects from the existing site(s) to the new site with no further action required.
+
+If you are switching from using bucket replication to using site replication, you must first remove all bucket replication rules from the healthy site prior to setting up site replication.
+
+Replication Resync
+------------------
+
+For scenarios where :ref:`bucket replication <minio-bucket-replication>` was in place prior to the failure, you can use :mc-cmd:`mc replicate resync` to restore data to a new site.
+Create a new site to replace the failed deployment, then synchronize the data from an existing bucket to the new site.
+
+1. Deploy a new MinIO site
+2. Set up IAM and users as needed
+3. On the site with data, create a new ``remote target`` using the :mc-cmd:`mc admin bucket remote add` command and record the ARN from the output
+4. From the site with the data, use the :mc-cmd:`mc replicate resync` command with the ARN from the previous command to rebuild the on the new site
+5. Wait for re-synchronization to complete (us :mc-cmd:`mc replicate resync status` to check)
+6. Set up bucket replication rule(s) from the new MinIO site to the existing target bucket(s)
+7. `(Optional)` Delete the bucket replication rules from the target deployment(s) to restore an active-passive replication scenario
+
+Bucket Replication
+------------------
+
+:ref:`Bucket replication <minio-bucket-replication>` can directly restore the site contents by performing a replication from the target bucket(s) to a new MinIO site.
+
+As a passive process, bucket replication may not perform as quickly as desired for a site recovery scenario.
+In such cases where performance is needed, use :mc-cmd:`mc replicate resync` described above instead.
```

Comment:
I think we're backing into the point here a little bit.

Using Bucket Replication in this way (post-disaster recovery) uses the standard replication scanner queue, which does not take priority over any other process (iirc). For recovery procedures with stricter SLA/SLO, use the active resync procedure.

Reply:
👍